wapuu.csv

```diff
@@ -134,4 +134,5 @@ wct2018-wapuu,ロックわぷー,https://github.com/momonotsubo/rockwapuu/blob/m
 fujisan-wapuu,Fujisan Wapuu,https://github.com/takeshi81/fujisan-wapuu/,https://www.meetup.com/ja-JP/Shizuoka-WordPress-Meetup/,https://raw.githubusercontent.com/takeshi81/fujisan-wapuu/master/fujisan-wapuu.png,image/png,Takeshi Igarashi,https://indigo-design.org/,Wapuu with Mt. Fuji: Made for Shizuoka WordPress Meetup
 fukuiraptor-wapuu,フクイラプトルわぷー,https://github.com/FukuiWPMeetup/fukuiraptorwapuu/,https://www.meetup.com/ja-JP/Fukui-WordPress-Meetup/,https://raw.githubusercontent.com/FukuiWPMeetup/fukuiraptorwapuu/master/fukuiraptor_wapuu.png,image/png,山森美穂,,Wapuu with Fukuiraptor created for Fukui WordPress Meetup
 akabeko-wapuu,赤べこわぷー,https://github.com/ayanoaizu/akabeko-wapuu/,,https://github.com/ayanoaizu/akabeko-wapuu/blob/master/akabeko-wapuu.png,image/png,ayanoaizu,https://github.com/ayanoaizu,Fukushima WordPress MeetUp vol.3
-wapuussoa,Wapuussoa,https://github.com/pedro-mendonca/Wapuussoa,https://2019.lisboa.wordcamp.org/,https://raw.githubusercontent.com/pedro-mendonca/Wapuussoa/master/images/2019.lisboa.wordcamp.org-wapuussoa.png,image/png,Pedro Mendonça,https://pedromendonca.pt,Wapuussoa - WordCamp Lisboa 2019
\ No newline at end of file
+wapuussoa,Wapuussoa,https://github.com/pedro-mendonca/Wapuussoa,https://2019.lisboa.wordcamp.org/,https://raw.githubusercontent.com/pedro-mendonca/Wapuussoa/master/images/2019.lisboa.wordcamp.org-wapuussoa.png,image/png,Pedro Mendonça,https://pedromendonca.pt,Wapuussoa - WordCamp Lisboa 2019
+atlanta-wapuu-2019,Atlanta Wapuu 2019,https://2019.atlanta.wordcamp.org/,https://github.com/sbddesign/atlanta-wapuu,https://raw.githubusercontent.com/sbddesign/atlanta-wapuu/master/2019/WordCamp-Atlanta-2019-Wapuu.png,image/png,Stephen DeLorme,https://d.elor.me,Wapuu for WordCamp Atlanta 2019
```

Comment:
The third column is a repository URL field, and the forth is wapuu URL field. You'll need to fix these :)

Reply:
Sorry! I've fixed it in my fork. I will close this and open a new pull request.